app/assets/javascripts/discourse/app/templates/modal/sidebar-section-form.hbs

```diff
@@ -91,8 +54,37 @@
       @icon="plus"
       @label="sidebar.sections.custom.links.add"
     />
-    {{#if this.currentUser.staff}}
-      <div class="row-wrapper">
+    {{#if this.activeSecondaryLinks}}
+      <hr />
+      <h3>{{I18n "sidebar.sections.custom.more_menu"}}</h3>
+      {{#each this.activeSecondaryLinks as |link|}}
+        <Sidebar::SidebarSectionFormLink
+          @link={{link}}
+          @deleteLink={{this.deleteLink}}
+          @reorderCallback={{this.reorder}}
+        />
+      {{/each}}
+      <DButton
+        @action={{action "addSecondaryLink"}}
+        @class="btn-flat btn-text add-link"
+        @title="sidebar.sections.custom.links.add"
+        @icon="plus"
+        @label="sidebar.sections.custom.links.add"
```

Comment:
I think we need to set `@ariaLabel` too. Kind of annoying that the component does not automatically take care of this.